### ISSUE TITLE
use Netz-AG design template

### ISSUE THREAD

**Issue by yuwash** [waschag-tvk/pywaschedv#2]
It seems Netz-AG wants us to use the common house webdesign. I don’t have access to the design and therefore can’t say how much effort it would be to switch.

**Comment by kyeap** [comment]
Some links at the welcome page has to be fixed. 
- WaschZoo to refer to welcome page if not logged in? 
- rules, documents and satzung

**Comment by yuwash** [comment]
You’re right, that page should be probably completely remade. It’s based on a copy of the old system… Feel free to change it as you like. Rules etc. could be added as [static files](https://docs.djangoproject.com/en/1.11/howto/static-files/). The best way would be probably to use some [CMS](https://djangopackages.org/grids/g/cms/), but that might be overkill for now.
